flight/targets/revolution/bl/pios_board.c

```diff
@@ -72,11 +72,13 @@ void PIOS_Board_Init()
 			break;
 		}
 	}
-	PIOS_Assert(flash_chip_ok);
-
 
 	PIOS_Flash_Internal_Init(&pios_internal_flash_id, &flash_internal_cfg);
 
+	if (!flash_chip_ok) {
+		pios_external_flash_id = 0;
```

Comment:
I would say `NULL`, but this is still an int? :(

Reply:
Yep